docs/faq.rst

```diff
@@ -75,3 +75,25 @@ If it crashes with a UnicodeError, what can I do?
 
         export LANG=en_US.UTF-8  # or similar, important is correct charset
 
+Why was borg forked from Attic?
+    borg was created in may 2015 in response to the difficulty of
+    getting new merge requests and larger changes incorporated into
+    attic. more details can be found in the `fairly long discussion
+    <https://github.com/jborg/attic/issues/217>`_ in 
+    attic issue queue that led to the fork.
+
+    borg intends to be:
+
+    * simple:
+      * as simple as possible, but no simpler
+      * do the right thing by default, but offer options
+    * open:
+      * welcome feature requests
+      * accept pull requests of good quality and coding style
+      * give feedback on PRs that can't be accepted "as is"
+      * discuss openly, don't work in the dark
+    * changing:
+      * do not break compatibility accidentally, without a good reason
```

Comment:
that was deliberate. i wanted to consolidate what was clear... i also think it's implicit in line 96 above.